public/src/components/Results.tsx

```diff
@@ -86,8 +87,8 @@ const Results = ({ review }: { review: boolean }) => {
             className: `${getSubmissionCellColor(submission, owner)} ${isSelected ? "selected" : ""} ${willBeReleased ? "release" : ""} ${pending ? "pending-review" : ""}`,
             onClick: () => {
                 actions.setSelectedSubmission(submission)
-                if (owner instanceof Enrollment) {
-                    actions.setActiveEnrollment(owner.clone())
+                if (owner.$typeName === "qf.Enrollment") {
```

Comment:
@JosteinLindhom Can this also use `isMessage(owner, EnrollmentSchema)`? If so, you should do a search for `"qf."` or `$typeName ===` and check for other places where you could use it... I didn't comment on all cases where we do string-based comparisons.

Reply:
Yes, converted all instances of `$typeName ===` now.